debian/crest-prov.install

```diff
@@ -0,0 +1,3 @@
+src/metaswitch/crest/tools /usr/share/clearwater/crest-prov/src/metaswitch/crest/
+src/metaswitch/crest/tools/sstable_provisioning/* /usr/share/clearwater/crest-prov/tools/sstable_provisioning
```

Comment:
I thought about that but decided that it would look weird for the scripts to be in crest and the venv in crest-prov. I'll do the docs update.